packages/mdc-textfield/_mixins.scss

```diff
@@ -63,36 +63,67 @@
   }
 }
 
+@mixin mdc-text-field-bottom-line-color($color) {
+  &:not(.mdc-text-field--disabled) {
+    @include mdc-text-field-bottom-line-color_($color);
+  }
+}
```

Comment:
move all the bottom-line mixins to bottom-line/_mixins.scss (and then export them through this file)